classes/Media.php

```diff
@@ -329,7 +323,7 @@ public static function getJqueryUIPath($component, $theme, $checkDependencies)
                 }
             }
         }
-        if (@filemtime($fileUri) || (defined('_PS_HOST_MODE_') && @filemtime($fileUriHostMode))) {
+        if (@filemtime($fileUri)) {
```

Comment:
Doesn't it mean `$fileUriHostMode` should be cleaned as well if it's only used for Host mode?

Reply:
Fixed